infra/cifuzz/http_utils.py

```diff
@@ -81,10 +81,11 @@ def get_json_from_url(url):
   Returns:
     A dictionary deserialized from JSON or None on failure.
   """
-  response = requests.get(url)
+  response = requests.get(url, timeout=10)
   try:
```

Comment:
I think `requests.get(url, timeout=10)` should be placed inside the `try`-block to ensure the `requests.exceptions.ReadTimeout` exception is caught.
```suggestion
  try:
    response = requests.get(url, timeout=10)
```

Reply:
Definitely. dumb mistake on my part!

Reply:
Thanks!

Reply:
A force push made the applied suggestion disappear, so this is still relevant at the time of writing.

Reply:
Commit e408199d8bac2da6de39b14b7db36a18820413a8 looks good to me. :)